statemachine/contrib/diagram.py

```diff
@@ -61,10 +61,12 @@ def _initial_edge(self):
 
     def _state_actions(self, state):
         entry = ", ".join(
-            [getattr(action.func, "__name__", action.func) for action in state.enter]
+            [getattr(action.func, "__name__", action.func)
+             for action in state.enter]
         )
         exit = ", ".join(
-            [getattr(action.func, "__name__", action.func) for action in state.exit]
+            [getattr(action.func, "__name__", action.func)
+             for action in state.exit]
```

Comment:
nitpick: These changes should be reverted to the library style (`line-length = 99`). If you run the configured linters they'll be automatically reverted.